e2e-tests/spring-graphql/src/test/kotlin/org/evomaster/e2etests/spring/graphql/base/GQLBaseCEEMTest.kt

```diff
@@ -0,0 +1,41 @@
+package org.evomaster.e2etests.spring.graphql.base
+
+import com.foo.graphql.base.BaseController
+import com.foo.graphql.base.BaseCustomEndpointController
+import org.evomaster.core.EMConfig
+import org.evomaster.e2etests.spring.graphql.SpringTestBase
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+
+class GQLBaseCEEMTest : SpringTestBase() {
+
+    companion object {
+        @BeforeAll
+        @JvmStatic
+        fun init() {
+            initClass(BaseCustomEndpointController())
+        }
+    }
+
+
+    @Test
+    fun testRunEM() {
+        runTestHandlingFlakyAndCompilation(
+                "GQL_BaseCEEM",
+                "org.foo.graphql.BaseCEEM",
+                20
+        ) { args: MutableList<String> ->
+
+            args.add("--problemType")
+            args.add(EMConfig.ProblemType.GRAPHQL.toString())
```

Comment:
in new tests, we can remove this, as this should be handled now automatically

Reply:
Done